app/services/content-service/server.js

```diff
@@ -16,14 +16,22 @@ import { makeServer, autoCaughtRouteError } from '../common';
 import * as endpoints from '../../shared/constants/endpoints';
 import { BUILT_UI_DIR } from '../../shared/paths-util';
 
-const VALID_PAGES = /(history|stars)/;
+const VALID_PAGES = [
+  'history',
+  'stars',
+];
```

Comment:
No, credits is auto generated and served as a separate file that has nothing to do with index.html. Can't rewrite that in react.